src/BaGet.Core/Mirror/FakeMirrorService.cs

```diff
@@ -9,6 +11,19 @@ namespace BaGet.Core.Mirror
     /// </summary>
     public class FakeMirrorService : IMirrorService
     {
+        Task<IReadOnlyList<string>> emptyVersions = Task.Factory.StartNew(() => new List<string>() as IReadOnlyList<string>);

```

Comment:
Instead of `Task.Factory.StartNew`, how about `Task.FromResult`?

Reply:
Will do.